src/api/ForForeldre.ts

```diff
@@ -0,0 +1,65 @@
+export interface TextPictureCard {
+  title: string;
+  text: string;
+  imgURL: URL;
+}
+export interface ForForeldreContent {
+  title: string;
+  ingress: string;
+  cards: Array<TextPictureCard>;
+  bottomText: string;
+}
+
+export default (): ForForeldreContent => {
+  return {
+    title: "Informasjon for foreldre",
+    ingress:
+      "Lurer du som forelder på mer om Vektorprogrammet? Her finner du informasjon om våre assistenter, kvalitetssikring, og foreldrekurs.",
+    cards: [
+      {
+        title: "Assistentene",
+        text: `Et verv som assistent hos Vektorprogrammet er 
```

Comment:
This is more of a question:
Not sure, but should the  be replaced with a `'` since it's a string?

Reply:
Hmmm, yes might be, but i think 
![image](https://user-images.githubusercontent.com/56224255/112282643-939dd480-8c87-11eb-8dc5-ffb8ded54e88.png)
 is multiline and `'` is not. If the strings should be singleline it should be changed, but i think for the time being multiline is clearer.

Reply:
Aha. That makes sense :+1: